scripts/tasks/test-runner.ts

```diff
@@ -17,31 +17,16 @@ export const testRunner: Task = {
 
     // We could split this out into a separate task if it became annoying
     const publishController = await servePackages({});
-    await exec(`yarn add --dev @storybook/test-runner jest-junit`, execOptions);
-
-    // Note we could split this out into a separate task too. We sort of do by checking if the file exists
-    const testFilePathname = join(sandboxDir, 'test-runner-jest.config.js');
-    if (junitFilename && !(await pathExistsSync(testFilePathname))) {
-      await exec(`yarn test-storybook --eject`, execOptions);
-      const testFile = await readFile(testFilePathname, 'utf8');
-
-      const reporters = `reporters: [
-            'default',
-            ['jest-junit', {
-              outputDirectory: '${junitFilename}',
-              addFileAttribute: 'true',
-              usePathForSuiteName: 'true',
-              uniqueOutputName: 'true' },
-            ],
-          ],`;
-
-      const newTestFile = testFile.replace(/}\s*$/m, `${reporters}$&`);
-      await writeFile(testFilePathname, newTestFile);
-    }
+    await exec(`yarn add --dev @storybook/test-runner@0.6.4--canary.179.c8f28d5.0`, execOptions);
 
     const storybookController = await serveSandbox(builtSandboxDir, {});
 
-    await exec(`yarn test-storybook --url http://localhost:8001`, execOptions);
+    await exec(`yarn test-storybook --url http://localhost:8001 --junit`, {
+      ...execOptions,
+      env: {
+        JEST_JUNIT_OUTPUT_FILE: junitFilename,
+      },
+    });
```

Comment:
Can you try this out too?

```suggestion
    await exec(`yarn test-storybook --url http://localhost:8001 --junit ${junitFilename}`, {
      ...execOptions,
    });
```